packages/jest-haste-map/src/index.ts

```diff
@@ -727,6 +732,41 @@ class HasteMap extends EventEmitter {
       roots: options.roots,
     };
 
+    const watchmanConfigPath = path.join(options.rootDir, '.watchmanconfig');
+    if (existsSync(watchmanConfigPath)) {
+      try {
+        const configStr = readFileSync(watchmanConfigPath).toString().trim();
+        const watchmanConfig = configStr === '' ? {} : JSON.parse(configStr);
+        const watchSymlinks = Boolean(watchmanConfig['watch_symlinks']);
```

Comment:
can we spawn `watchman get-config options.rootDir` instead?

https://facebook.github.io/watchman/docs/get-config

Reply:
> The get-config command returns the .watchmanconfig for the root. If there is no .watchmanconfig, it returns an empty configuration field.

I think we have equivalent functionality here for a lower performance cost than spawning a new process

Reply:
Fair enough 🙂